.github/workflows/ci.yml

```diff
@@ -11,39 +11,74 @@ on:
 jobs:
   build:
     runs-on: macos-latest
+    env:
+      NOT_FORK: ${{secrets.NOT_FORK == 'true'}}
+      FORK: ${{secrets.NOT_FORK != 'true'}}
```

Comment:
You can remove FORK, I think, because you should be able to use boolean logic in Github Actions ifs. Like `!NOT_FORK`. But maybe it's then better to use FORK instead? Not not fork is confusing.

Reply:
`!NOT_FORK` doesn't work. already tried. Environment variables are treated as strings.
And, yeah I added `FORK` to reduce confusion from not of not.